guides/08_custom-components/06_frequently-asked-questions.md

```diff
@@ -1,7 +1,14 @@
 # Frequently Asked Questions
 
 ## What do I need to install before using Custom Components?
-Before using Custom Components, make sure you have Python 3.8+, Node.js v16.14+, npm 9+, and Gradio 4.0+ installed.
+Before using Custom Components, make sure you have Python 3.10+, Node.js v16.14+, npm 9+, and Gradio 4.0+ installed.
```

Comment:
lets bump this to node 18, i don't think it is needed but better we do it now than later and 16 is pretty old now (22 is current).